Extension/src/LanguageServer/client.ts

```diff
@@ -1775,7 +1775,7 @@ class DefaultClient implements Client {
     private readonly referencesProgressDelayInterval: number = 2000;
 
     private reportReferencesProgress(progress: vscode.Progress<{message?: string; increment?: number }>, forceUpdate: boolean): void {
-        const helpMessage: string = this.model.referencesCommandMode.Value === ReferencesCommandMode.Peek ? "" : " Click the search icon to preview results.";
+        const helpMessage: string = this.model.referencesCommandMode.Value === ReferencesCommandMode.Peek ? "" : " Click the search icon in the status bar to preview results.";
```

Comment:
is it possible to use the actual search icon in the text?

Reply:
We can't use the same icon in the text as the one that shows up in the status bar.  The unicode character renders differently too.

Reply:
We can if we use the search Unicode character, but it looks "odd" with colors. The status bar has a special image.